server/routes/prisonerSelect/prisonerSelect.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from 'express'
+import { FormError } from '../../@types/template'
+import PrisonerSearchService, { PrisonerSearchSummary } from '../../services/prisonerSearchService'
+
+type PageData = {
+  error?: FormError
+  searchResults?: PrisonerSearchSummary[]
+}
+
+export default class PrisonerSelectRoutes {
+  constructor(private readonly prisonerSearchService: PrisonerSearchService) {}
+
+  private renderView = async (req: Request, res: Response, pageData: PageData): Promise<void> => {
+    const { error, searchResults } = pageData
+
+    return res.render('pages/prisonerSelect', {
+      errors: error ? [error] : [],
+      searchResults,
+    })
+  }
+
+  view = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const searchTerm = JSON.stringify(req.query.searchTerm)
+
+    if (!searchTerm) return res.redirect('/search-for-prisoner')
```

Comment:
**Not blocking**

We define route paths at a higher level, but then the routes can still know directly what paths exist.
It might be worth defining paths in centralised constants that can be referenced.

What do you think?

Reply:
Yeah I think we should do this, will look to do it in a future PR.